MC/bin/o2_hybrid_gen.py

```diff
@@ -63,6 +63,18 @@ def main():
                         "iniFile": ""
                     }
                 })
+            elif gen == "evtpool":
+                gens.append({
+                    'name': 'evtpool',
+                    'config': {
+                        "eventPoolPath": "/path/to/filename.root",
+                        "skipNonTrackable": True,
+                        "roundRobin": False,
+                        "randomize": True,
+                        "rngseed": 0,
+                        "randomphi": False,
+                    }
+                })
```

Comment:
I guess this is fine for now. In future, we could deduce these fields dynamically using the ROOT dictionaries. Python can surely simply iterate over the class fields.